File: VHS/AP CS A/ICT/03-ICT/CoinAmounts.c
/*
 *  CoinAmounts.c
 *  
 *
 *  Created by Daniel McGuigan on 19.11.10.
 *  Copyright 2010 __MyCompanyName__. All rights reserved.
 *
 */

#include "CoinAmounts.h"

void findChange(int change){
	printf("%d", change);
	
	int q, d, n, p;
	q = change/25;
	d = (change % 25)/10;
	n = ((change % 25) % 10)/5;
	p = (((change % 25) % 10) % 5)/1;
	
	printf("\n %d", q);
	printf("\n %d", d);
	printf("\n %d", n);
	printf("\n %d \n", p);
}

File: VHS/AP CS A/ICT/03-ICT/CoinAmounts.java
public class CoinAmounts
{
    private int myChange;
    
    public CoinAmounts (int change)
    {
        myChange = change;
    }
    
    public void findChange()
    {
        System.out.println(myChange + " cents =>");
        int quarters = myChange / 25;
        System.out.println("Quarters = \t" + quarters);
        int dimes = (myChange % 25) / 10;
        System.out.println("Dimes = \t" + dimes);
        int nickels = ((myChange % 25) % 10) / 5;
        System.out.println("Nickels = \t" + nickels);
        int pennies = (((myChange % 25) % 10) % 5) / 1;
        System.out.println("Pennies = \t" + pennies);
        System.out.println();
    }
}